src/Pages/Home/ColumnsComponents/ColumnList.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import styled from "styled-components";
+import { flexCenter } from "../../../Styles/Theme";
+import { FaIndustry } from "react-icons/fa";
+import { BsGeoAlt } from "react-icons/bs";
+import { RiBankLine, RiLightbulbLine } from "react-icons/ri";
+import { MdWork } from "react-icons/md";
+import Column from "./Column";
+
+const ColumnList = (props) => {
+  const { title, list, id } = props.columData;
+
+  return (
+    <ColumnListContainer>
+      <TitleContainer>
+        {id === 0 && <RiLightbulbLine size="30" color="white" />}
+        {id === 1 && <BsGeoAlt size="30" color="white" />}
+        {id === 2 && <FaIndustry size="30" color="white" />}
+        {id === 3 && <MdWork size="30" color="white" />}
+        {id === 4 && <RiBankLine size="30" color="white" />}
```

Comment:
```suggestion
const idObj = {
  0: <RiLightbulbLine size="30" color="white" />,
  1: <BsGeoAlt size="30" color="white" />,
  2: <FaIndustry size="30" color="white" />,
  3: <MdWork size="30" color="white" />,
  4: <RiBankLine size="30" color="white" />
}
//
        {idObj[id]}
```
컴포넌트 바깥에 해당 아이콘들을 맵핑한 객체를 만들어주고 컴포넌트 안에서는 그 객체에 키값을 넘겨주는 방식을 사용해도 되겠네요. 하지만 지금 방식도 너무 깔끔합니다.

Reply:
멘토님 방식으로 수정 하였습니다.